src/components/Pages/Location/LocationInfo.js

```diff
@@ -131,7 +131,14 @@ const LocationPageFacilityHours = ({ hours }) => {
               </td>
               <td>
                 {hours[day] !== null
-                  ? hours[day]
+                  ? hours[day].split(',').map( (dateRange, i) => {
+                    const comma = i+1 < hours[day].split(',').length ? "," : ""
+                    return (
+                      <div class="coa-LocationPage__hours-section">
+                        {dateRange}{comma}&nbsp;
+                      </div>
+                    )
+                  })
```

Comment:
This was a bit tricky. What we're doing here is adding a comma `,` AND additional `<div>` for dates with multiple ranges. I racked my head on a clearer way to do this, but wasn't finding the muse🧚‍♂️.

Reply:
It took me a second, but I get what's going on here. Because if we're not the last dateRange, then we need to add a comma. I think it works.